src/pages/VenuePage/VenuePage.tsx

```diff
@@ -228,9 +228,12 @@ export const VenuePage: React.FC = () => {
     return <LoadingPage />;
   }
 
-  const { entrance, template, hasPaidEvents } = venue;
+  const { template, hasPaidEvents } = venue;
 
-  const hasEntrance = Array.isArray(entrance) && entrance.length > 0;
+  const hasEntrance =
+    world?.entrance &&
+    Array.isArray(world.entrance) &&
+    world.entrance.length > 0;
```

Comment:
Shoudln't this be simplified like above? 
```suggestion
  const hasEntrance =  !!world?.entrance?.length;
```

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/2546/commits/08a3290e454507ee3163e87d8f4078506f8d7b8f